Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectListViewController.swift

```diff
@@ -10,9 +10,14 @@ import UIKit
 
 class ProjectListViewController: UIViewController {
 
+    var currentUser: UserModel = UserModel()
+    
+    @IBOutlet weak var tempLabel: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        if currentUser.firstName != nil {
```

Comment:
Xcode gives a Swift compiler error for this line: "Comparing non-optional value of type 'String' to nil always returns true"
This is exactly how I would write this in other languages but apparently in Swift, strings can never be nil unless they are defined as optional. Since firstName is set as empty in UserModel.swift you can change this line to: `if !currentUser.firstName.isEmpty {` to remove this error. I'm not sure I'm familiar enough with optional variables quite yet to know the best way to resolve this, or if these variables are just temporary to support this sprint and none of this really matters too much in the long run. Thoughts?